site/_resources/faq.md

```diff
@@ -27,4 +27,21 @@ The HTTPProxy custom resource is an attempt to solve these issues with an API th
 More importantly, the HTTPProxy CRD is designed with inclusion in mind, a feature that enables administrators to configure top-level ingress settings (for example, which virtual hosts are available to each team), while delegating the lower-level configuration (for example, the mapping between paths and backend services) to each development team.
 More information about the HTTPProxy API can be found [in the HTTPProxy documentation][1].
 
+# Q: When I load my site in Safari, it shows me an empty page. Developer tools show that the HTTP response was 421. Why does this happen?
```

Comment:
Should we add the version tested of Safari? Presuming eventually in the future the issue gets resolved?

Reply:
I can't be sure about the versions, so I'd prefer to update this if we ever find out there is a version that fixes the problem.